HW9/task1.py

```diff
@@ -0,0 +1,46 @@
+class Company:
+    def __init__(self, name: str, founded: str, ceo: str, location: str, employees: int, capital: int = None,
+                 nasdaq: str = None):
+        self.__name = name.capitalize()
+        self.founded = founded
+        self.ceo = ceo
+        self.location = location
+        self.employees = employees
+        self.__capital = capital
+        self.nasdaq = nasdaq
```

Comment:
why are all these vars public?

Where are properties with limitations for them?